<launch>
 <node pkg="map_tool"    type="point_publisher"   name="point_publisher" output="screen">
   <param name="path"       value="/home/joseph/param/"/>
   <param name="map_frame"  value="map"/>
   <param name="subscribe_topic" value="floor"/>
   <param name="matching_pose_topic" value="ndt_pose"/>
   <param name="wating_time" value="2"/>

 </node>
</launch>